prefetch.bs

```diff
@@ -396,3 +396,16 @@ The <dfn>list of sufficiently strict speculative navigation referrer policies</d
 
     <div class="issue">Further review is needed of the cookies-present handling and its interaction with redirects and caches.</div>
 </div>
+
+<div algorithm>
+    To <dfn export>prefetch</dfn> given a {{Document}} |document|, [=URL=] |url| and [=referrer policy=] |referrerPolicy|, perform the following steps.
+
+    1. Let |partitionKey| be the result of [=determining the network partition key=] given |document|'s [=relevant settings object=].
+    1. Let |topLevelOrigin| be |document|'s [=relevant settings object=]'s [=environment/top-level origin=] if |document|'s [=Document/browsing context=] is not a [=top-level browsing context=], and |url|'s [=origin=] otherwise.
```

Comment:
This line is a bit confusing because of the negative. Maybe

> Let topLevelOrigin be url's origin if document's browsing context is a top-level browsing context; otherwise let it be document's relevant settings object's origin.

With that formulation I think I can see the logic more clearly: if we're navigating a TLBC, the _new_ top-level origin will be related to url, whereas if we're navigating a nested BC, the top-level origin will stay the same.

Reply:
Done.